src/control.rs

```diff
@@ -1,5 +1,6 @@
 //! A couple of functions to enable and disable coloring.
 
+use once_cell::sync::Lazy;
```

Comment:
I'm assuming the other crates used in the library have `extern crate` definitions in the root because of an old Rust version that's being targeted, but I'll have to double check once the crate gets an MSRV figured out (see #85).

This would technically be a [breaking change](https://github.com/colored-rs/colored/pull/133#issuecomment-1616709253), and I'd probably end up putting this inside a v3 of `colored`, which is already planned if #139 ends up happening. I'll probably end up making a `v2` branch or something, and then I can get this merged in.